### PR TITLE
gocqlx/table/Table: support timestamp at the granularity of batch statements

### PR DESCRIPTION
**Pull request by GingerMoon**
I'd like to add some public functions to expose CRUD query builders,  which is similar as
func (t *Table) SelectBuilder(columns ...string) *qb.SelectBuilder {
to enable timestamp at the granularity of batch statements.

### CODE REVIEW

**Comment by GingerMoon** [comment]
Hi @mmatczuk do you know who I should contact for reviewing this PR?

**Comment by mmatczuk** [comment]
Hi @GingerMoon thanks for pinging me.

File: table/table.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type Table struct {
 
 	get    cql
 	sel    cql
-	insert cql
```
Comment (mmatczuk) [inline table/table.go]: I'd rather we did not change that.

File: table/table.go [hunk 2]
```diff
@@ -100,15 +97,30 @@ func (t *Table) SelectBuilder(columns ...string) *qb.SelectBuilder {
 
 // Insert returns insert all columns statement.
 func (t *Table) Insert() (stmt string, names []string) {
-	return t.insert.stmt, t.insert.names
+	return t.InsertBuilder().ToCql()
+}
+
+// InsertBuilder returns a builder initialised to insert all columns statement.
+func (t *Table) InsertBuilder() *qb.InsertBuilder {
```
Comment (mmatczuk) [inline table/table.go]: The builders are mutable you cannot return a pointer like that.

It should be implemented as UpdateBuilder and DeleteBuilder.

**Pull request merged**